rest-client/src/main/java/com/gentics/mesh/parameter/client/SortingParametersImpl.java

```diff
@@ -1,28 +1,6 @@
 package com.gentics.mesh.parameter.client;
 
-import java.util.LinkedHashMap;
-import java.util.Map;
-import java.util.function.Function;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-
-import com.gentics.mesh.core.rest.SortOrder;
 import com.gentics.mesh.parameter.SortingParameters;
 
 public class SortingParametersImpl extends AbstractParameters implements SortingParameters {
-
-	private final Map<String, SortOrder> sort = new LinkedHashMap<>();
```

Comment:
As the sorting can be used over several fields, including the ones not supported in the query line, this content has to be back.

Reply:
It is used [below](https://github.com/gentics/mesh/pull/1532/files#diff-49eddd22b8e20004a0cbe175434b5123f168fa26fe86286b4a866d6600f409c7R68) as well.

Reply:
As mentioned in our talk: this class is only used with the MeshRestClient for passing parameters to requests and the removed code prevented this.